setup.py

```diff
@@ -7,10 +7,10 @@
 
 cur_dir = path.abspath(path.dirname(__file__))
 
-with open(path.join(cur_dir, 'requirements.txt')) as f:
+with open(path.join(cur_dir, 'requirements.txt', 'r')) as f:
```

Comment:
```suggestion
with open(path.join(cur_dir, 'requirements.txt'), 'rt') as f:
```

Just curious - why not `t` as well, since it's text?

Reply:
cause 'r' was the one that work at first and I went with it 😅